### PR TITLE
{bio}[intel/2019b] BSMAPz v1.1.1 w/ Python 2.7.16

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/BSMAPz/BSMAPz-1.1.1-intel-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+sanity_check_commands = [
+    # 'bsmapz -h' returns exit-code 1 - testing it with example data instead
+    'bsmapz -a %(builddir)s/%(name)s-%(version)s/test_data/simulated.fastq -z 33 -p 2 -q 20 '
+    '-d %(builddir)s/%(name)s-%(version)s/test_data/test.fasta -S 77345 -w 1000 '
+    '-o %(builddir)s/%(name)s-%(version)s/test_r1.bam',
+    'methdiff.py -h',
+    'methratio.py -h',
+]
+
+modextrapaths = {'PYTHONPATH': 'bin'}
```
Comment (boegel) [inline easybuild/easyconfigs/b/BSMAPz/BSMAPz-1.1.1-intel-2019b.eb]: @Darkless012 Why is this needed? I don't see any `import` statements in the scripts in `bin` that confirm this is really needed?

The Python scripts have executable permissions, and `bin/` is added to `$PATH` automatically, that should suffice?

Comment (Darkless012) [reply easybuild/easyconfigs/b/BSMAPz/BSMAPz-1.1.1-intel-2019b.eb]: True, will change that. Thanks.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3131.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e9bf32f89abd4dde5e58d85e8e13fa69 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/f8d20ae75850da9737c7abad20b9886d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!
